cwl/src/main/scala/cwl/ontology/Schema.scala

```diff
@@ -106,7 +126,15 @@ object Schema {
   private def loadOntologyFromIri(ontologyManager: OWLOntologyManager)(schemaIri: String): ErrorOr[OWLOntology] = {
     validate {
       val iri = IRI.create(schemaIri)
-      ontologyManager.loadOntologyFromOntologyDocument(iri)
+      ontologyCache.flatMap(_.getIfPresent(iri) |> Option.apply) match {
+        case Some(ontology) =>
+          ontologyManager.copyOntology(ontology, OntologyCopy.DEEP)
```

Comment:
[Codecov](https://github.com/codecov/browser-extension#readme) says this isn't tested. Will need a unit test for this line.